nibabel/processing.py

```diff
@@ -310,3 +311,80 @@ def smooth_image(img,
                                    mode=mode,
                                    cval=cval)
     return out_class(sm_data, img.affine, img.header)
+
+
+def conform(from_img,
+            out_shape=(256, 256, 256),
+            voxel_size=(1.0, 1.0, 1.0),
+            order=3,
+            cval=0.0,
+            orientation='RAS',
+            out_class=Nifti1Image):
```

Comment:
From the docstring, the default behavior is to keep the input class. I think that's sensible.

```suggestion
            out_class=None):
```